packages/react-router-dev/vite/plugin.ts

```diff
@@ -938,6 +972,7 @@ export const reactRouterVitePlugin: ReactRouterVitePlugin = () => {
     let reactRouterBrowserManifest: ReactRouterManifest = {
       ...fingerprintedValues,
       ...nonFingerprintedValues,
+      sri: undefined,
```

Comment:
I think this is intended to avoid shipping the sri map to the browser but I don't think it's working quite right?  

![Screenshot 2025-03-06 at 11 03 28 AM](https://github.com/user-attachments/assets/4d36d2f4-e41a-4f0f-97d4-446dfae6560c)

Reply:
removed